docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/client/ClientBuilderForPlugin.java

```diff
@@ -15,6 +15,7 @@
 public class ClientBuilderForPlugin  {
 
     private final DockerClientConfig config;
+    private Integer readTimeout;
```

Comment:
hm, some timeout existed before in configuration but not used AFAIR, requires review...